OpenRA.Game/Sound/Sound.cs

```diff
@@ -115,6 +115,8 @@ ISound Play(SoundType type, Player player, string name, bool headRelative, WPos
 			if (string.IsNullOrEmpty(name) || DisableAllSounds || (DisableWorldSounds && type == SoundType.World))
 				return null;
 
+			// TODO: Should this allow sounds when controlling allied units?
+			// (Do we even ever provide a non-null player?)
```

Comment:
This probably get passed `RenderPlayer` (or even `LocalPlayer`) somewhere which might or might not be null.